projects/1.20/assets/goldelight/goldelight/lang/zh_cn.json

```
@@ -0,0 +1,313 @@
+  "item.goldelight.ancient_gold_ingot": "远古金锭",
+  "item.goldelight.ancient_gold_nugget": "远古金块",
+  "item.goldelight.ancient_gold_upgrade_smithing_template": "锻造模板",
+  "item.goldelight.cooked_golden_bacon": "熟金培根",
+  "item.goldelight.cooked_golden_chicken_cuts": "熟金鸡肉丁",
+  "item.goldelight.cooked_golden_cod_slice": "熟金鳕鱼片",
+  "item.goldelight.cooked_golden_mutton_chops": "熟金羊排",
+  "item.goldelight.cooked_golden_rice": "熟金稻米饭",
+  "item.goldelight.cooked_golden_salmon_slice": "熟金鲑鱼片",
+  "item.goldelight.fried_golden_egg": "煎金鸡蛋",
+  "item.goldelight.fried_golden_rice": "炒金稻米饭",
+  "item.goldelight.gapple_cider": "金苹果酒",
+  "item.goldelight.golden_apple_pie_slice": "金苹果派切片",
+  "item.goldelight.golden_bacon": "金培根",
+  "item.goldelight.golden_bacon_and_eggs": "金培根鸡蛋",
+  "item.goldelight.golden_bacon_sandwich": "金培根三明治",
+  "item.goldelight.golden_baked_cod_stew": "金烘焙鳕鱼炖",
+  "item.goldelight.golden_barbecue_stick": "金烧烤串",
+  "item.goldelight.golden_bee_spawn_egg": "金蜜蜂刷怪蛋",
+  "item.goldelight.golden_beef": "金牛肉",
+  "item.goldelight.golden_beef_patty": "金牛肉饼",
+  "item.goldelight.golden_beef_stew": "金牛肉炖",
+  "item.goldelight.golden_bone": "金骨头",
+  "item.goldelight.golden_bread": "金面包",
+  "item.goldelight.golden_broth": "金大骨汤",
+  "item.goldelight.golden_cabbage": "金卷心菜",
+  "item.goldelight.golden_cabbage_leaf": "金卷心菜叶",
+  "item.goldelight.golden_cabbage_rolls": "金卷心菜卷",
+  "item.goldelight.golden_cabbage_seeds": "金卷心菜种子",
+  "item.goldelight.golden_cake": "金蛋糕",
+  "item.goldelight.golden_cake_slice": "金蛋糕切片",
+  "item.goldelight.golden_chicken": "金鸡",
+  "item.goldelight.golden_chicken_cuts": "金鸡肉丁",
+  "item.goldelight.golden_chicken_sandwich": "金鸡肉三明治",
+  "item.goldelight.golden_chicken_soup": "金鸡汤",
+  "item.goldelight.golden_chicken_spawn_egg": "金鸡刷怪蛋",
+  "item.goldelight.golden_chocolate_pie_slice": "金巧克力派切片",
+  "item.goldelight.golden_cocoa_beans": "金可可豆",
+  "item.goldelight.golden_cod": "金鳕鱼",
+  "item.goldelight.golden_cod_roll": "金鳕鱼寿司",
+  "item.goldelight.golden_cod_slice": "金鳕鱼片",
+  "item.goldelight.golden_cookie": "金曲奇",
+  "item.goldelight.golden_cow_spawn_egg": "金牛刷怪蛋",
+  "item.goldelight.golden_dog_food": "金狗粮",
+  "item.goldelight.golden_egg": "金鸡蛋",
+  "item.goldelight.golden_egg_sandwich": "金鸡蛋三明治",
+  "item.goldelight.golden_feather": "金羽毛",
+  "item.goldelight.golden_fish_stew": "金鱼肉炖",
+  "item.goldelight.golden_fruit_salad": "金水果沙拉",
+  "item.goldelight.golden_glazed_ham": "盘装金火腿",
+  "item.goldelight.golden_glow_berries": "金发光浆果",
+  "item.goldelight.golden_glow_berry_custard": "金发光浆果蛋奶沙司",
+  "item.goldelight.golden_ham": "金火腿",
+  "item.goldelight.golden_hamburger": "金汉堡包",
+  "item.goldelight.golden_honey_bottle": "金蜂蜜瓶",
+  "item.goldelight.golden_honey_cookie": "金蜂蜜曲奇",
+  "item.goldelight.golden_honeycomb": "金蜜脾",
+  "item.goldelight.golden_horse_feed": "金马食",
+  "item.goldelight.golden_hot_cocoa": "金热可可",
+  "item.goldelight.golden_ink_pasta": "金鱿鱼意面",

```

```suggestion
  "item.goldelight.golden_ink_pasta": "金墨鱼汁意面",
```
不知道译者吃没吃过那种会爆浆的煮墨鱼，老香了

---

农夫乐事译法，作为农夫乐事附属还是需要遵照原模组的翻译

---

没吃过这种的
修改为金鱿鱼墨面，与农夫乐事中一致